### PR TITLE
Adds manpage and fixes several spelling issue found by lintian

### PR DESCRIPTION
**Pull request by blattms**
This is actually from 2021.04 but somehow fell through the cracks.

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this please

File: doc/man1/flow.1 [hunk 1]
```diff
@@ -0,0 +1,364 @@
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.8.
```
Comment (joakim-hove) [inline doc/man1/flow.1:1]: Maybe flow - as the flagship executable - deserves a bit more prose in the man page?

**Comment by akva2** [comment]
abusing this for some testing.

**Comment by akva2** [comment]
jenkins run volve update_volve_data please

**Comment by joakim-hove** [comment]
> jenkins run volve update_volve_data  

Ehh - since the VOLVE dataset is unfortunately not 100% free we have discontinued the process of integrating the VOLVE dataset: https://github.com/opm/opm-tests/pull/672

**Comment by akva2** [comment]
slack.

**Comment by akva2** [comment]
jenkins run volve update_volve_data please

**Comment by akva2** [comment]
jenkins run volve update_volve_data please

**Comment by akva2** [comment]
jenkins run extra update_extra_data please

**Comment by blattms** [comment]
On second thought the manpage proably needs an update because of new options (thousands of it :wink:)

File: doc/man1/flow.1 (hunk 1 above)
Comment (blattms) [reply doc/man1/flow.1:1]: Printready recommendations for descriptions are welcome.

File: doc/man1/flow.1 [hunk 2]
```diff
@@ -117,20 +126,23 @@ The minimum number of Newton iterations per time step used by flow. Default: 1
...
-.TP
 \fB\-\-ignore\-keywords\fR=\fI\,STRING\/\fR
 List of Eclipse keywords which should be ignored. As a ':' separated string. Default: ""
 .TP
 \fB\-\-ilu\-fillin\-level\fR=\fI\,INTEGER\/\fR
 The fill\-in level of the linear solver's ILU preconditioner. Default: 0
 .TP
 \fB\-\-ilu\-redblack\fR=\fI\,BOOLEAN\/\fR
-Use red\-black partitioning for the ILU preconditioner. Default: false
+Use red\-black partioning for the ILU preconditioner. Default: false
```
Comment (bska) [inline doc/man1/flow.1]: How did this slip through original review?

File: doc/man1/flow.1 [hunk 3]
```diff
@@ -180,11 +195,17 @@ Maximum time step size where single precision floating point arithmetic can be u
...
 Maximum number of iterations to determine solution the  well equations. Default: 30
 .TP
+\fB\-\-maximum\-number\-of\-well\-switches\fR=\fI\,INTEGER\/\fR
+Maximum number of times a well can switch to the same control. Default: 3
+.TP
 \fB\-\-milu\-variant\fR=\fI\,STRING\/\fR
-Specify which variant of the modified\-ILU preconditioner ought to be used. Possible variants are: ILU (default, plain ILU), MILU_1 (lump diagonal with dropped row entries), MILU_2 (lump diagonal with the sum of the absolute values of the dropped row  entries), MILU_3 (if diagonal is positive add sum of dropped row entrires. Otherwise subtract them), MILU_4 (if diagonal is positive add sum of dropped row entrires. Otherwise do nothing. Default: "ILU"
+Specify which variant of the modified\-ILU preconditioner ought to be used. Possible variants are: ILU (default, plain ILU), MILU_1 (lump diagonal with dropped row entries), MILU_2 (lump diagonal with the sum of the absolute values of the dropped row  entries), MILU_3 (if diagonal is positive add sum of dropped row entrires. Otherwise substract them), MILU_4 (if diagonal is positive add sum of dropped row entrires. Otherwise do nothing. Default: "ILU"
```
Comment (bska) [inline doc/man1/flow.1:208]: Adds a misprint ("subtract" gets replaced by "substract").

**Review by bska — COMMENTED** [review]
Since the text here was generated by `help2man` the new misprints are not the result of manual work.  Still, we should fix them before we add the manual pages.

File: doc/man1/flow.1 (hunk 3 above)
Comment (blattms) [reply doc/man1/flow.1:208]: Thanks.
Shoot. These kind of errors drive my crazy. Did not use a flow with the current changes of #3699 for generating the man-page.

Of course I already have built 2021.10-1 packages and uploaded them to [mentors.debian.net](https://mentors.debian.net/package/opm-simulators/):
```
 I typo-in-manual-page

    usr/share/man/man1/flow.1.gz partioning partitioning
    usr/share/man/man1/flow.1.gz substract subtract
```

**Review by bska — APPROVED** [review]
Thanks for the updates.  This looks good to me now.  I'll just run one more build check and merge if that's successful.

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
> I'll just run one more build check and merge if that's successful.

Build check is green as expected so I'll merge this now.  Thanks a lot!

**Comment by blattms** [comment]
Thanks.

**Pull request merged**